ts/store/reducers/content.ts

```diff
@@ -81,19 +95,26 @@ export const contextualHelpDataSelector = (
   state: GlobalState
 ): pot.Pot<ContextualHelp, Error> => state.content.contextualHelp;
 
+export const idpsSelector = createSelector(
+  contentSelector,
+  (content: ContentState): ReadonlyArray<SpidIdp> =>
+    isReady(content.idps) ? content.idps.value.items : idps
+);
+
 /**
  * return an option with Idp contextual help data if they are loaded and defined
  * @param id
  */
-export const idpContextualHelpDataFromIdSelector = (id: IdentityProviderId) =>
+export const idpContextualHelpDataFromIdSelector = (id: SpidIdp["id"]) =>
   createSelector<GlobalState, pot.Pot<ContextualHelp, Error>, Option<Idp>>(
     contextualHelpDataSelector,
     contextualHelpData =>
       pot.getOrElse(
         pot.map(contextualHelpData, data => {
           const locale = getLocalePrimaryWithFallback();
-          return fromNullable(
-            data[locale] !== undefined ? data[locale].idps[id] : undefined
+
+          return fromNullable(data[locale]).fold(none, l =>
+            fromNullable(l.idps[id as IdentityProviderId])
```

Comment:
```suggestion
          return fromNullable(data[locale]).chain(l =>
            fromNullable(l.idps[id as IdentityProviderId])
          );
```
In order to make more readable.